cucumber/authenticators_oidc/features/support/authn_oidc_helper.rb

```diff
@@ -27,8 +28,16 @@ def set_id_token_user_property_variable
     set_oidc_variable("id-token-user-property", oidc_id_token_user_property)
   end
 
-  def set_oidc_variable(variable_name, value)
-    path = "cucumber:variable:conjur/authn-oidc/keycloak"
+  def set_provider_uri_variable_without_service_id(value = oidc_provider_uri)
```

Comment:
This doesn't exactly apply here since you're not setting an instance variable.  Still, I agree that `set_` is not as precise as it could be.  I also consider `variable` to be a legacy naming mistake, and try not to perpetuate it.  With these two in mind I'd suggest either:

```
create_oidc_secret
```
or
```
create_oidc_variable
```